diff --git a/Configurations/10-main.conf b/Configurations/10-main.conf
index a497bcc..08314ee 100644
--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -818,6 +818,8 @@ my %targets = (
     # specifications,
     "linux64-riscv64" => {
         inherit_from     => [ "linux-generic64"],
+        cflags           => add("-march=rv64gcv"),
+        cxxflags         => add("-march=rv64gcv"),
         perlasm_scheme   => "linux64",
         asm_arch         => 'riscv64',
     },
